docs/cheatsheet.rst

```diff
@@ -2,12 +2,70 @@
 Cheatsheet
 **********
 
-.. index:: operator; precedence
+.. index:: precedence
```

Comment:
Even if we're ultimately going to revert it, I'd keep the new index entry (and with `!` if we restore the table here).